commands/lib/install.go

```diff
@@ -75,3 +75,44 @@ func installLibrary(lm *librariesmanager.LibrariesManager, libRelease *libraries
 	taskCB(&rpc.TaskProgress{Message: "Installed " + libRelease.String(), Completed: true})
 	return nil
 }
+
+//ZipLibraryInstall FIXMEDOC
+func ZipLibraryInstall(ctx context.Context, req *rpc.ZipLibraryInstallReq) (*rpc.ZipLibraryInstallResp, error) {
+	res := &rpc.ZipLibraryInstallResp{}
+	lm := commands.GetLibraryManager(req.GetInstance().GetId())
+	Path := req.GetPath()
+	if err := installZipLibrary(lm, Path); err != nil {
```

Comment:
We don't need the `installZipLibrary`, call directly `LibrariesManager.InstallZipLib`.

Reply:
The normal Library install procedure is divided into two functions `LibraryInstall()` and `installLibrary()` so I thought I'd follow the same style. As it does not actually have a unique function, I'll remove it asap.

Reply:
Interesting, the normal procedure will probably need to be changed to if it's like that, not in this PR though.